BookingsApi/BookingsApi.UnitTests/Controllers/JudiciaryParticipantsControllerTests.cs

```diff
@@ -0,0 +1,36 @@
+using BookingsApi.Controllers.V1;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Queries.Core;
+using BookingsApi.Domain.Validations;
+using BookingsApi.Infrastructure.Services.IntegrationEvents;
+
+namespace BookingsApi.UnitTests.Controllers;
+
+
+public class JudiciaryParticipantsControllerTests
```

Comment:
I did write an integration test initially, but I can't get it to throw a valid domain rule exception naturally without mocking it, and get the coverage on the 'throw;'. If I try to set the hearing to cancelled or something, itll catch that issue too early and return a 404 (probably because how the query is written. So needs it needs a mock to guide it into the right place for the sonar coverage